README.rst

```diff
@@ -1,69 +1,105 @@
-===========  =================================================================================================================================
-Info         Contains a Python API for interacting with NI-DAQmx. See `GitHub <https://github.com/ni/nidaqmx-python/>`_ for the latest source.
-Author       National Instruments
-===========  =================================================================================================================================
++------------+-----------------------------------------------------------+
+| **Info**   | Contains a Python API for interacting with NI-DAQmx. See  | 
+|            | `GitHub <https://github.com/ni/nidaqmx-python/>`_ for the |
+|            | latest source.                                            |
++------------+-----------------------------------------------------------+
+| **Author** | National Instruments                                      |
++------------+-----------------------------------------------------------+
+
+.. contents:: Table of Contents
+   :depth: 1
+   :backlinks: none
 
 About
 =====
 
 The **nidaqmx** package contains an API (Application Programming Interface)
-for interacting with the NI-DAQmx driver. The package is implemented in Python.
-The package is implemented as a complex,
-highly object-oriented wrapper around the NI-DAQmx C API using the
+for interacting with the NI-DAQmx driver.
+
+Documentation
+-------------
+
+You can find the latest API documentation for the **nidaqmx** package on
+`Read the Docs <http://nidaqmx-python.readthedocs.io/en/latest>`_.
+
+Refer to the `NI-DAQmx Help <http://digital.ni.com/express.nsf/bycode/exagg4>`_
+for API-agnostic information about NI-DAQmx or measurement concepts. NI-DAQmx
```

Comment:
I think "and" would work better than "or" here.

Is "API-agnostic" necessary? It's unclear whether it refers to NI-DAQmx only or both NI-DAQmx and measurement concepts.